ksql-engine/src/test/java/io/confluent/ksql/integration/JsonFormatTest.java

```diff
@@ -51,9 +59,6 @@
   private TopicProducer topicProducer;
   private TopicConsumer topicConsumer;
 
-  private Map<String, GenericRow> inputData;
-  private Map<String, RecordMetadata> inputRecordsMetadata;
-
   @ClassRule
   public static final EmbeddedSingleNodeKafkaCluster CLUSTER = new EmbeddedSingleNodeKafkaCluster();
```

Comment:
What do the changes in this test have to do with the ticket?

Reply:
I copied my new test from this one, so refactored this one a little in the process. Could of gone in another PR...